frontend/src/metabase/actions/containers/ActionCreator/CreateActionForm/CreateActionForm.tsx

```diff
@@ -36,66 +27,36 @@ const ACTION_SCHEMA = Yup.object({
 });
 
 type FormValues = Pick<
-  CreateQueryActionOptions,
+  CreateQueryActionParams,
   "name" | "description" | "model_id"
 >;
 
 interface OwnProps {
-  question: Question;
-  formSettings: ActionFormSettings;
-  modelId?: CardId;
-  onCreate?: (values: WritebackQueryAction) => void;
+  initialValues?: Partial<FormValues>;
+  onCreate: (values: FormValues) => void;
   onCancel?: () => void;
 }
 
-interface DispatchProps {
-  handleCreateAction: (
-    action: CreateQueryActionOptions,
-  ) => Promise<WritebackQueryAction>;
-}
-
-type Props = OwnProps & DispatchProps;
-
-const mapDispatchToProps = {
-  handleCreateAction: Actions.actions.create,
-};
+type Props = OwnProps;
```

Comment:
nit: 
```suggestion
type CreateActionFormProps = OwnProps;
```

To stay consistent with other declarations e.g. `type ActionCreatorProps = OwnProps & StateProps & DispatchProps;`